src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -44,92 +129,175 @@ const ApplicationModal = props => {
         </Modal>
       ) : (
         <Modal
-          title="Application"
+          title="Review Application"
           visible={displayModal}
           onOk={handleOk}
           onCancel={handleCancel}
           afterClose={handleCancel}
           className="modalStyle"
-          footer={null}
+          footer={[
+            <Button key="back" onClick={handleCancel}>
+              Return to Previous
+            </Button>,
+            /**
+             * Author: Khaleel Musleh
+             * @param {onSubmit={approveApplication}} e Added an onSubmit Handler once the admin presses the approve button.
+             */
+            <Button key="submit" type="primary" onSubmit={approveApplication}>
+              Approve
+            </Button>,
+            <Popconfirm
+              /**
+               * Author: Khaleel Musleh
+               * @param {onSubmit={rejectApplication}} e Added an onSubmit Handler once the admin presses the reject button.
+               */
+
+              title="Are you sure you want to reject?"
+            >
+              <Button key="submit" onSubmit={rejectApplication} danger>
```

Comment:
Hey Khaleel, for ant design, we need to use onConfirm instead of onSubmit here to be able to run the rejectApplication once we click the confirm button from the pop up that says are you sure you want to reject this application?

Reply:
@BrittanyPete It should display the pop up for the onConfirm, so it should show the pop up then the buttons of reject or cancel, ive changed the onSubmit to onConfirm.

Reply:
@BrittanyPete Ive changed both to onConfirm with the last 2 commits I did, if you can please check them and if everything is as expected.